bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+log = logging.getLogger(__name__)
+
+
+class PrideLeader(commands.Cog):
+    """Gives information about Pride Leaders."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.pride = self.load_pride_leader_json()
+
+    @staticmethod
+    def load_pride_leader_json() -> dict:
+        """Loads pride leader information from static json resource."""
+        explanation_file = Path("bot/resources/pride/prideleader.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            pride = json.load(json_data)
+
+        return pride
+
+    def name_verifier(self, leader_name: str) -> str:
```

Comment:
The function also returns `None` when leader_name is not in self.pride. Need to update function return type
```suggestion
    def name_verifier(self, leader_name: str) -> Optional[str]:
```